app/models/course/condition/level.rb

```diff
@@ -1,4 +1,8 @@
 class Course::Condition::Level < ActiveRecord::Base
   acts_as_condition
   validates :minimum_level, numericality: { greater_than: 0 }
+
+  def title
+    I18n.t('activerecord.models.attributes.course.condition.level/title', value: minimum_level)
```

Comment:
I spent a while playing with this, and I found a few things:
 - I18n should not be directly used inside ActiveRecord models.
 - Use `human_attribute_name`; in this case, it's `human_attribute_name('title.title', value: minimum_level)`
 - The I18n key for this is

```yml
en:
  activerecord:
    attributes:
      course/condition/level/title:
        title: 'Level %{value}'
```